cli/generate/src/render.rs

```diff
@@ -1483,6 +1565,11 @@ impl Generator {
             add_line!(self, ".field_map_slices = ts_field_map_slices,");
             add_line!(self, ".field_map_entries = ts_field_map_entries,");
         }
+        if !self.supertype_map.is_empty() && self.abi_version >= ABI_VERSION_WITH_SUPERTYPE_INFO {
+            add_line!(self, ".supertype_map_slices = ts_supertype_map_slices,");
+            add_line!(self, ".supertype_map_entries = ts_supertype_map_entries,");
+            add_line!(self, ".supertypes = ts_supertypes,");
```

Comment:
I think calling this field `supertype_symbols` would be clearer, and then the corresponding const array `ts_supertype_symbols`